configs/build_targets.yaml

```diff
@@ -0,0 +1,17 @@
+# build_targets.yaml
+# Maps node names to their build scripts and config files.
+# Add new nodes as needed.
+
+streamdiffusion:
+  script: /workspace/ComfyUI/custom_nodes/ComfyUI-StreamDiffusion/scripts/build_tensorrt_engines.py
```

Comment:
This is neat, does it support arguments for other scripts like build_trt.py?

We will need to cover the case when an image is built using nodes.yaml, and entrypoint.sh is ran with `--build-engines`, Just need a basic check to not attempt build if the custom nodes folder or script doesn't exist.

Reply:
Can cherry-pick this one to guard the other engine builds or refactor to your script

https://github.com/livepeer/comfystream/commit/979fcef5718eed4c5249782ed0694803b9239dc4

Reply:
> This is neat, does it support arguments for other scripts like build_trt.py?
> 
> We will need to cover the case when an image is built using nodes.yaml, and entrypoint.sh is ran with `--build-engines`, Just need a basic check to not attempt build if the custom nodes folder or script doesn't exist.

The [entrypoint.sh]already checks for the existence of build_targets.yaml, the required folder, and the script before attempting StreamDiffusion engine builds (see lines 142–144 and the Python block around line 150+). It skips the build if any are missing, just like the new logic for other nodes which you referred for cherry picking.